BUILD

```diff
@@ -59,7 +59,7 @@ cc_library(
         "upb/decode.int.h",
         "upb/encode.c",
         "upb/msg.c",
-        "upb/msg.h",
+        "upb/msg.int.h",
```

Comment:
at some point we should rename `int` to `internal`.  It took me quite a while to figure out what it meant and filenames are practically free.

Reply:
Sounds good, I'll follow up with a change that does this for all of the `.int.h` files.